mkosi.default.d/10-systemd.conf

```diff
@@ -6,6 +6,8 @@
 Format=gpt_btrfs
 Bootable=yes
 HostonlyInitrd=yes
+# Prevent ASAN warnings when building the image
+Environment=ASAN_OPTIONS=verify_asan_link_order=false
```

Comment:
I wonder why it should be set to `false` here? If `mkosi` ends up using `systemd` libraries built and installed with ASan without setting `LD_PRELOAD=` it seems it should be fixed either by passing on `LD_PRELOAD=` somewhere or by avoiding using half-installed `systemd` to build images.

Reply:
mkosi itself is fine, it's running dracut that causes issues. We can't set LD_PRELOAD in mkosi itself because we'd leak context from systemd into mkosi. Ignoring the sanitizer errors during image build is probably fine. It's only set during the build process and everything from boot onwards is using LD_PRELOAD.

Reply:
> Ignoring the sanitizer errors during image build is probably fine

Got it. I think as long as it works it should be fine.